Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -300,27 +298,14 @@ def FractionRing :=
 namespace FractionRing
 
 instance unique [Subsingleton R] : Unique (FractionRing R) :=
-  Localization.unique
+  Localization.instUniqueLocalizationToCommMonoid
 #align fraction_ring.unique FractionRing.unique
 
 instance [Nontrivial R] : Nontrivial (FractionRing R) :=
   ⟨⟨(algebraMap R _) 0, (algebraMap _ _) 1, fun H =>
       zero_ne_one (IsLocalization.injective _ le_rfl H)⟩⟩
 
-variable {A}
-
-noncomputable instance : Field (FractionRing A) :=
-  { Localization.commRing,
-    IsFractionRing.toField A with
-    add := (· + ·)
-    mul := (· * ·)
-    neg := Neg.neg
-    sub := Sub.sub
-    one := 1
-    zero := 0
-    nsmul := AddMonoid.nsmul
-    zsmul := SubNegMonoid.zsmul
-    npow := Localization.npow _ }
+noncomputable instance : Field (FractionRing A) := IsFractionRing.toField A
```

Comment:
Is there a reason why this golf wasn't used in ML3?

Reply:
I've added a porting note explaining why this instance was golfed.

Reply:
There was no reason. ML3 builds fine with this change. I will assume we don't want to backport it.